doc/data/messages/n/nan-comparison/good.py

```diff
@@ -1 +1,3 @@
-# This is a placeholder for correct code for this message.
+x = 42
+if x == 42:  # [nan-comparison]
+    print("Bad!")
```

Comment:
```suggestion
import numpy as np

def both_nan(x, y) -> bool:
   return np.isnan(x) and np.isnan(y)
```

Reply:
functional tests fail if there's 2 linting failures on the same line so I just did one

Reply:
It's possible to have multiple messages raised on the same line in functional tests with ``# [message-1, message-2]``. This would also permit to illustrate that both math.isnan or np.isnan are okay in good.py 😄

Reply:
didn't know that! Thanks! will update